operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [CHANGE] Grafana Agent: Depreate `metaMonitoring.grafanaAgent` values. #12307
+  * Grafana Agent has been deprecated in earlier 2024, and will reach End-of-Support by the end of 2025.
+  * Instead of provisioning agent's Kubernetes resources with the chart's `metaMonitoring.grafanaAgent.enabled`, users are recommended to collect Mimir's meta-monitoring data with an external collector. We recommend Grafana k8s-monitoring, that manages the creation and lifecycle of Alloy instances, and has built-in support for collecting telemetry from Grafana LGTM stacks.
```

Comment:
```suggestion
  * Instead of provisioning agent's Kubernetes resources with the chart's `metaMonitoring.grafanaAgent.enabled`, collect Mimir's meta-monitoring data with an external collector. It's recommended to use Grafana k8s-monitoring, which manages the creation and lifecycle of Alloy instances and has built-in support for collecting telemetry from Grafana LGTM stacks.
```